src/zenml/cli/server.py

```diff
@@ -240,9 +241,17 @@ def up(
             cli_utils.declare(
                 f"The local ZenML dashboard is available at "
                 f"'{server.status.url}'. You can connect to it using the "
-                f"'{DEFAULT_USERNAME}' username and an empty password."
+                f"'{DEFAULT_USERNAME}' username and an empty password. To open the dashboard in a browser automatically, set the env variable AUTO_OPEN_DASHBOARD=true."
             )
 
+            if os.environ.get(ENV_AUTO_OPEN_DASHBOARD) == "true":
```

Comment:
Here I would expect the default behavior to be that the dashboard opens unless you explicitly set this env variable. In `zenml.constants` we have a util function which can do exactly that (and is more lenient in which values the user may set for the variable).

```suggestion
            if handle_bool_env_var(ENV_AUTO_OPEN_DASHBOARD, default=True):
```